### DISCUSSION TITLE
(paint.net) Switch to MSI installer

### DISCUSSION THREAD

**Discussion by marcinozga** [chocolatey-community/chocolatey-packages#1933] [Ideas]
Switch Paint.net to msi installer. Currently silent installation doesn't work as installer displays gui, so deploying it via psexec, ansible, etc. to remote PCs fails. Starting with version 4.3.3, paint.net developer publishes msi installers. See his comment.
https://forums.getpaint.net/topic/118566-feature-request-paintnet-exe-installer-silent-quiet-install/#comment-585769

Installers on github release page:
https://github.com/paintdotnet/release/releases

**Comment by AdmiringWorm** [comment]
Before opening this up, we need someone to be willing to take on the work of switching to the MSI installer.

Out of curiosity, to the MSI handle automatically upgrading a version of Paint.net that was previously installed with the `.exe`?

If not, that will need to be handled as well in the installation script (or rather, as a helper).
